referral_module/api/tests/test_campaigns_api.py

```diff
@@ -0,0 +1,104 @@
+from django.test import TestCase
+
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from referral_module.tests import factories
+from referral_module import models
+
+
+class CampaignsApiTestCase(TestCase):
+    def setUp(self):
+        # Every test needs a client.
+        self.user = User.objects.create_user('usertest', 'user@mail.com', 'liorlior')
+        self.client.login(username='usertest', password='liorlior')
+        self.campaign = []
```

Comment:
We've already spoken about this `self.campaign` is something for the singular, but you return the list of objects.